unit/java_bytecode/java_bytecode_parse_generics/RecursiveGeneric.java

```diff
@@ -1,4 +1,4 @@
-class RecursiveGeneric extends SimpleRecursiveGeneric<RecursiveGeneric>
+class RecursiveGeneric<T extends RecursiveGeneric<T>>
```

Comment:
This was supposed to be just renaming, I will correct this.

Reply:
This is actually correct, the original file `RecursiveGeneric` got renamed to `RecursiveGeneric_Derived`, and the original `SimpleRecursiveGeneric` got in turn renamed to `RecursiveGeneric`